assets/js/modules/analytics-4/datastore/accounts.js

```diff
@@ -43,24 +43,41 @@ import { createFetchStore } from '../../../googlesitekit/data/create-fetch-store
 import { actions as errorStoreActions } from '../../../googlesitekit/data/create-error-store';
 import { createValidatedAction } from '../../../googlesitekit/data/utils';
 import { isValidAccountSelection } from '../utils/validation';
+import { caseInsensitiveListSort } from '../../../util/case-insensitive-sort';
+import { appendAccountID, appendPropertyAndAccountIds } from '../utils/account';
 
 const { receiveError, clearError, clearErrors } = errorStoreActions;
 
 const fetchGetAccountSummariesStore = createFetchStore( {
 	baseName: 'getAccountSummaries',
-	controlCallback() {
+	controlCallback( params ) {
+		const { pageToken = '' } = params || {};
 		return API.get(
 			'modules',
 			'analytics-4',
 			'account-summaries',
-			{},
+			{
+				pageToken,
+			},
 			{
 				useCache: false,
 			}
 		);
 	},
-	reducerCallback( state, accountSummaries ) {
-		return { ...state, accountSummaries };
+	argsToParams: ( args ) => {
+		return { pageToken: args?.pageToken ?? '' };
```

Comment:
`argsToParams` is a way to transform positional args into an object of parameters since the signature of many actions uses one or more positional args.

I think we could do the same to simplify things for the account summaries fetch action since the only parameter it has that we'll use is the pageToken, so we could update it like so
```suggestion
	argsToParams: ( pageToken ) => {
		return { pageToken };
```
This would require a change where it's called of course but could help simplify things a bit 👍 I'm not against keeping it as an object though because sometimes positional args can be hard to understand in context.